app/views/transacs/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<section class="container1">
+    <% @transacs.each do |trans| %>
+        <div class="row border my-1">
+  <div class="col-9">
+
+    <p class="transac-name">Transaction <%= trans.name %></p>
+    <p class="date"><%= trans.created_at %></p>
+  </div>
+  <div class="col-3">
+    <span class="amount-dls">$<%= trans.amount %></span>
+  </div>
+</div>
+    <% end %>
+</section>
+
+<div class="bottom-container position-fixed bottom-0 w-100 btn-bg">
+  <div class="inner-bottom-container text-center">
+    <%= link_to "New transactions", new_transac_path, class: "text-white" %>
+  </div>
+</div>
```

Comment:
- [ ] The list of transactions should presented ordered by the most recent: Kindly fix it.
![image](https://user-images.githubusercontent.com/38635158/184506514-819425fc-9747-4694-831b-60930b134e6b.png)